src/core/redux/Store.tsx

```diff
@@ -9,6 +13,10 @@ export const initStore = (initialState: {} = {}) => {
     preloadedState: initialState,
     reducer: {
       articles: articlesReducer,
+      careerLocations: careerLocationsReducer,
+      careerOpportunities: careerOpportunitiesReducer,
+      companies: companiesReducer,
+      jobTypes: jobTypesReducer,
```

Comment:
What is the benefit/reason of 4 reducers instead of 1 career reducer? Are they being used differently / outside of career opportunities?

Reply:
The idea is keep all kinds of entities separate from each other. There is no absolute or correct way to handle this though.

My thoughts are that each kind of entity should be separated into each own reducer because they are re-used in other parts of the app or duplicated in this part.
E.g. each career opportunity is connected to a company, but a single company  may have many career opportunities. This would lead us to have duplicates of that company on our store.
If we put the companies in their own reducer we get to use the store as a bit of a cache, since when you go to the company details page from an opportunity page the company data is already populated in the store, and you won't have to wait for it to be fetched to view the page.

This could also let us change how data is fetached more easily. We could change the career api to not deliver company data at initial load.